### PR TITLE
[helm create] Include serviceAccount.annotations value

### PR DESCRIPTION
**Pull request by naseemkullah**
In order to ease the use of features such as [GKE Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster), it is beneficial to allow users to annotate service accounts out of the box.

### CODE REVIEW

**Comment by naseemkullah** [comment]
bump

**Comment by bacongobbler** [comment]
cc @helm/charts-maintainers.

**Comment by davidkarlsen** [comment]
I think this looks sensible!

File: pkg/chartutil/create.go [hunk 1]
```diff
@@ -103,6 +103,7 @@ fullnameOverride: ""
 serviceAccount:
   # Specifies whether a service account should be created
   create: true
+  annotations: {}
```
Comment (hickeyma) [inline pkg/chartutil/create.go]: I think it would be helpful to add a comment to the `annotations` value

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for adding this @naseemkullah. A small change needed to add a comment to the value.

File: pkg/chartutil/create.go (hunk 1 above)
Comment (naseemkullah) [reply pkg/chartutil/create.go]: True, I was not sure if it was self explanatory enough or not. Thanks, added!

**Comment by naseemkullah** [comment]
My pleasure @hickeyma. The change has been applied.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks for this @naseemkullah.

Going to merge following feedback from chart maintainer @davidkarlsen.

**Pull request merged**